src/app/_components/FilterContainer.tsx

```diff
@@ -6,10 +6,17 @@ type LayoutProps = {
   children: ReactNode
 }
 
-type FilterProps = {
+type DesktopFilterProps = {
   search: ReactNode
+  location?: ReactNode
```

Comment:
Oei - I'm passing something as specific as `location` here slightly weird, it's like the component knows too much?

Reply:
I agree, it could be a nice improvement from what we have.

This means reworking the blog, events and ecosystem explorer pages. How about we keep it for a different PR?

Reply:
I think we misunderstood each other - It shouldn't be part of this PR .. this PR can be merged last.

Reply:
I implemented the changes here: https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/903

Reply:
Fixed by https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/903